### ISSUE TITLE
define search templates

### ISSUE THREAD

**Issue by moonSandra** [elastic/elasticsearch-dsl-py#1947]
hi. i have a question. is there anything that we could define search templates like as a class in elasticsearch_dsl?

**Comment by miguelgrinberg** [comment]
No, this is not available. But you can create a function that builds a query using arguments, and this in a way provides the same functionality.
